middleware/dnstap/out/tcp_test.go

```diff
@@ -0,0 +1,66 @@
+package out
+
+import (
+	"net"
+	"testing"
+)
+
+func sendOneTcp(tcp *TCP) error {
+	if _, err := tcp.Write([]byte("frame")); err != nil {
+		return err
+	}
+	if err := tcp.Flush(); err != nil {
+		return err
+	}
+	return nil
+}
+func TestTcp(t *testing.T) {
+	tcp := NewTCP("localhost:14000")
+
+	if err := sendOneTcp(tcp); err == nil {
+		t.Fatal("not listening but no error")
```

Comment:
Test messages are different from errors, as they will not be wrapped and will be directly read by humans. So make them a full sentence "Not listening, but no error."
Why is this an Fatal in the test btw?